{{cookiecutter.project_name}}/requirements-dev.txt

```diff
@@ -1,20 +1,21 @@
 -r requirements.txt
-black==20.8b1
-factory-boy==3.2.0
-flake8==3.9.0
-flake8-blind-except==0.2.0
-flake8-debugger==3.2.1
+black==22.3.0
+click==8.1.3
```

Comment:
AFAIK we don't use `click` explicitly. Would be nice to have a comment here about why it has to be added to the requirements as well as the conditions that have to be met in order to be able to remove it again.

Reply:
Ups, it isn't needed. I forgot to remove it.

Reply:
:+1: same in `Makefile`.

Reply:
Ups, thanks. :sweat_smile:

Reply:
Oh wait, pipeline fails. That's why it's needed there. Otherwise we install a lower version of ```click```, which doesn't work with newer ```black``` versions

Reply:
I did add one in the ```Makefile```.